.github/workflows/all-post-commit-workflows.yaml

```diff
@@ -11,11 +11,20 @@ permissions:
   contents: read
   pages: write
   id-token: write
+  packages: write
 
 jobs:
   static-checks:
     uses: ./.github/workflows/all-static-checks.yaml
     secrets: inherit
+  build-wheels:
+    needs: build-artifact
+    uses: ./.github/workflows/eager-package-main.yaml
+    secrets: inherit
+  build-docker-with-wheels-artifact:
```

Comment:
Run an experiment to test performance of just installing the wheel before test execution. If acceptable, remove this step.

Reply:
Waiting ....